docs/parseparam.rst

```diff
@@ -0,0 +1,43 @@
+#This document includes the details on the all the parse-able params in Quokka.
+#These parameters are read in the readParameters() function in simulation.hpp.
+
+
+---------------------------------------------------------------------------—————————————————————————-
```

Comment:
Can you change this to use ReStructured Text formatting: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html?